src/styles/catalog/cards.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 300ms ease-in-out;
```

Comment:
it's not a good practice to use `all`. Specify what rules you'd like to change exactly. Check similar cases as well